src/components/core/base/VisualizationLoadingHOC.tsx

```diff
@@ -56,6 +74,10 @@ export interface IVisualizationLoadingState {
     isLoading: boolean;
 }
 
+interface IVisualizationLoadingProps {
+    sdk?: SDK;
```

Comment:
Could this be added to ICommonVisualizationProps? Then you can remove it from ICommonChartProps.

Reply:
done